templates_tests/interface_with_logrus_test.go

```diff
@@ -104,3 +114,25 @@ func TestTestInterfaceWithLogrus_F(t *testing.T) {
 		assert.Nil(t, errorMessage)
 	})
 }
+
+type ContextHook struct {
+}
+
+func (hook *ContextHook) Fire(entry *logrus.Entry) error {
+	if entry.Context != nil {
+		entry.Data["has_context"] = entry.Context.Value(key)
+	}
+	return nil
+}
+
+func (hook *ContextHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.TraceLevel,
+		logrus.DebugLevel,
+		logrus.InfoLevel,
+		logrus.WarnLevel,
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
+	}
+}
```

Comment:
Based on the fact the context.Context var is always named "ctx" can we just get rid of this and check that there's no "ctx" var in the callingRecord above?